fearless/CoreLayer/OperationFactory/NFT/AlchemyNFTOperationFactory.swift

```diff
@@ -188,10 +264,69 @@ final class AlchemyNFTOperationFactory {
             }
         }
     }
+
+    // MARK: NFTs for collection
+
+    private func createFetchNftsForCollectionOperation(
+        address: String,
+        url: URL,
+        offset: Int
+    ) -> BaseOperation<AlchemyNftsResponse> {
+        let authorizedUrl = url.appendingPathComponent(ThirdPartyServicesApiKeysDebug.alchemyApiKey)
```

Comment:
why debug?

Reply:
Because we use it for all alchemy requests. I don't know the reason.